src/ralph/defaults.py

```diff
@@ -107,6 +109,16 @@ def load_config(config_file_path):
     },
 }
 
+# Backends
+
+DATABASE_BACKENDS = [backend.value for backend in DatabaseBackends]
+PARSERS = [parser.value for parser in Parsers]
```

Comment:
Could you place it elsewhere as it is not backends ?

Reply:
Good Idea! Thanks) This has been updated in the #204 PR)